spec/unit/actions/synchronize_users_spec.rb

```diff
@@ -130,5 +130,28 @@
         let(:message_for_project) { :import_users }
       end
     end
+
+    context 'when using unsuported mode' do 
+      let(:params) do
```

Comment:
Similar code found in synchronize_users_spec.rb:47, synchronize_users_spec.rb:76, synchronize_users_spec.rb:135 (mass = 60)